src/MyArrayList.java

```diff
@@ -0,0 +1,325 @@
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f4");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f5");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f6");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f7");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f8");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f9");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        System.out.println("add 8s8");
+        obj.add(8, "8s8");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+        obj.add("f10");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("add11");
+        obj.add(11, "f11");
+        obj.printAll();
+
+        System.out.println("add8");
+        obj.add(8, "s8");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("set4");
+        System.out.println(obj.set(4, "s4"));
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("set19");
+        obj.set(19, "s19");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("set20");
+        obj.set(20, "s20");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        obj.add(21, "f21");
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("remove19");
+        obj.remove(19);
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        System.out.println("remove18");
+        obj.remove(18);
+        obj.printAll();
+        System.out.println("count = " + obj.count);
+
+        obj.printAll();
+
+
+        int x;
+        x = obj.indexOf("f6");
+        System.out.println("indexOf отработал: " + x);
+        System.out.println(obj.contains("f23"));
+
+        System.out.println("indexOf отработал: " + obj.indexOf("f100500"));
+        System.out.println("indexOf отработал: " + obj.indexOf(null));
+
+//        obj.remove("f7");
+//        obj.remove("f13");
+//        System.out.println("count = " + obj.count);
+
+        System.out.println(obj.remove(7));
+        System.out.println(obj.remove(12));
+        System.out.println("count = " + obj.count);
+
+        obj.printAll();
+
+
+        /*
+        System.out.println(obj.contains("f2"));
+
+        obj.clear();
+        System.out.println(obj.get(0));
+        System.out.println("count = " + obj.count);
+        */
+
+
+        Iterator iterator = obj.iterator();
+        while (iterator.hasNext()) {
+            System.out.println(iterator.next());
```

Comment:
Please add test where you remove each 4th element using iterator. Because current tests insufficent